CodingChallenges/CC_135_GIF_Loop/P5/sketch.js

```diff
@@ -0,0 +1,40 @@
+// GIF Loop
+// Daniel Shiffman
+// https://thecodingtrain.com/CodingChallenges/135-gif-loop.html
+// https://youtu.be/nBKwCCtWlUg
+
+const totalFrames = 120;
+let counter = 0;
+const record = false;
+
+function setup() {
+  createCanvas(400, 400);
+}
+
+function draw() {
+  let percent = 0;
+  if (record) {
+    percent = float(counter) / totalFrames;
+  } else {
+    percent = float(counter % totalFrames) / totalFrames;
+  }
+  render(percent);
+  if (record) {
+    save("output/gif-" + nf(counter, 3) + ".png");
+    if (counter == totalFrames - 1) {
+      exit();
```

Comment:
```suggestion
      noLoop();
```

`exit()` doesn's apply for p5. Also it might be worth investigating https://p5js.org/reference/#/p5/saveFrames? Eventually I'd like to make an example with https://github.com/wenheLI/p5.gif/!

Reply:
>Also it might be worth investigating https://p5js.org/reference/#/p5/saveFrames? Eventually I'd like to make an example with https://github.com/wenheLI/p5.gif/!

I'll take a look.